Makefile

```diff
@@ -80,30 +81,30 @@ publish-repo:
     --type docker
 
 serve-repo:
-	LUET_NOLOCK=true $(LUET) serve-repo --port 8000 --dir $(ROOT_DIR)/build
+	LUET_NOLOCK=true $(LUET) serve-repo --port 8000 --dir $(DESTIONATION)
```

Comment:
```suggestion
	LUET_NOLOCK=true $(LUET) serve-repo --port 8000 --dir $(DESTINATION)
```

Reply:
Oh man, I solved the conflicts on Makefile too fast :sweat: